packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -250,32 +245,94 @@ export function getPublicInstance(instance: Instance): * {
   return instance;
 }
 
-export function prepareForCommit(containerInfo: Container): void {
+function createEvent(type: TopLevelType): Event {
+  const event = document.createEvent('Event');
+  event.initEvent(((type: any): string), false, false);
+  return event;
+}
+
+function dispatchBeforeDetachedBlur(target: HTMLElement): void {
+  const targetInstance = getClosestInstanceFromNode(target);
```

Comment:
If both flags are off, can we exit early so this function can be fully DCE?

Reply:
This function only gets called from where there are flags. It gets removed from the output without needing the flags here.